documentation/modules/ROOT/pages/configuration/storage.adoc

```diff
@@ -0,0 +1,705 @@
+|`debezium_database_history`
+|The name of the table to which the history will be stored.
+
+|[[deprecated-jdbc-history-schema-history-table-ddl]]<<deprecated-jdbc-history-schema-history-table-ddl, `schema.history.internal.jdbc.schema.history.table.ddl`>>
+|<<jdbc-history-schema-history-table-ddl-default, `see table section`>>
+|The DDL statement used to create the storage table.
+
+|[[deprecated-jdbc-history-schema-history-table-select]]<<deprecated-jdbc-history-schema-history-table-select, `schema.history.internal.jdbc.schema.history.table.select`>>
+|<<jdbc-history-schema-history-table-select-default, `see table section`>>
+|The `SELECT` statement to read the internal schema history form the table.
+
+|[[deprecated-jdbc-history-schema-history-table-exists-select]]<<deprecated-jdbc-history-schema-history-table-exists-select, `schema.history.internal.jdbc.schema.history.table.exists.select`>>
+|<<jdbc-history-schema-history-table-exists-select-default, `see table section`>>
+|The `SELECT` statement to check the existence of the storage table.
+
+|[[deprecated-jdbc-history-schema-history-table-insert]]<<deprecated-jdbc-history-schema-history-table-insert, `schema.history.internal.jdbc.schema.history.table.insert`>>
+|<<jdbc-history-schema-history-table-insert-default, `see table section`>>
+|The `INSERT` statement to record the schema history to the table.
+
+|===
+
+=== History Table defaults
+
+[[jdbc-history-schema-history-table-ddl-default]]<<jdbc-history-schema-history-table-ddl-default,1. Table Structure>>
+[SQL]
+----
+CREATE TABLE %s (
+    id VARCHAR(36) NOT NULL,
+    history_data VARCHAR(65000),
+    history_data_seq INTEGER,
+    record_insert_ts TIMESTAMP NOT NULL,
+    record_insert_seq INTEGER NOT NULL
+)
+----
+
+[[jdbc-history-schema-history-table-select-default]]<<jdbc-history-schema-history-table-select-default,2. Select Query>>
+[SQL]
+----
+SELECT id, history_data, history_data_seq FROM %s
+    ORDER BY record_insert_ts, record_insert_seq, id, history_data_seq
+----
+
+[[jdbc-history-schema-history-table-exists-select-default]]<<jdbc-history-schema-history-table-exists-select-default,4. Data exist Query>>
+[SQL]
+----
+SELECT * FROM %s LIMIT 1
+----
+
+[[jdbc-history-schema-history-table-insert-default]]<<jdbc-history-schema-history-table-insert-default,4. Insert Query>>
+[SQL]
+----
+INSERT INTO %s(id, history_data, history_data_seq, record_insert_ts, record_insert_seq) VALUES ( ?, ?, ?, ?, ? )
+----
+
+== Redis
+
+The store uses Redis cache to store the data using https://redis.io/docs/latest/develop/clients/jedis/[Jedis client].
+
+=== Offset Store
+
+[cols="35%a,10%a,55%a",options="header"]
+|===
+|Property
+|Default
+|Description
+
+|[[offset-storage-redis]]<<offset-storage-redis, `offset.storage`>>
+|No default
+|Must be set to `io.debezium.storage.redis.offset.RedisOffsetBackingStore`
```

Comment:
If this and similar properties have required values, can they be coded to carry the appropriate defaults?